src/swsssdk/dbconnector.py

```diff
@@ -9,10 +9,76 @@
 # FIXME: Convert to metaclasses when Py2 support is removed. Metaclasses have unique interfaces to Python2/Python3.
 
 class SonicDBConfig(object):
+    SONIC_DB_GLOBAL_CONFIG_FILE = "/var/run/redis/sonic-db/database_global.json"
     SONIC_DB_CONFIG_FILE = "/var/run/redis/sonic-db/database_config.json"
+    _sonic_db_config_dir = "/var/run/redis/sonic-db"
+    _sonic_db_global_config_init = False
     _sonic_db_config_init = False
     _sonic_db_config = {}
 
+    """This is the database_global.json parse and load API. This file has the namespace name and
+       the corresponding database_config.json file. The global file is significant for the
+       applications running in the linux host namespace, like eg: config/show cli, snmp etc which
+       needs to connect to databases running in other namespaces. If the "namespace" attribute is not
+       specified for an "include" attribute, it refers to the linux host namespace.
+       If the user passes namespace parameter, this API loads json file for that namespace alone.
+    """
+    @staticmethod
+    def load_sonic_global_db_config(global_db_file_path=SONIC_DB_GLOBAL_CONFIG_FILE, namespace=None):
+        """
+        Parse and load the global database config json file
+        """
+        if SonicDBConfig._sonic_db_global_config_init == True:
+            return
+
+        if os.path.isfile(global_db_file_path) ==  True:
```

Comment:
>global_db_file_path [](start = 26, length = 19)

If global_db_file_path does not exist, why set _sonic_db_global_config_init to True?

Also can you return _sonic_db_global_config_init value to caller?